CPS-0004/README.md

```diff
@@ -0,0 +1,96 @@
+6. While it may be useful to draw inspiration from existing real-world systems, it should also be understood that blockchain governance will have an entirely different set of constraints.  For example, the ADA community does not share one culture or geographic locality. Additionally, the decisions being made are not the same kinds of decisions that a world government needs to make.
+7. The uncontrolled hard fork described above is dangerous (from a safety standpoint) and disruptive (from an economic standpoint); it will likely seriously undermine the cohesion of the Cardano community. For this reason, it should be considered a tool of absolute last resort.
+
+## **Questions a Solution Ought Answer**
+
+In this section, based on the goals above, we outline an incomplete set of questions that any proposed solution likely ought to answer:
+
+1. What are the exact categories of actors considered by the proposed system, and what actions are available to them under what conditions? What are the explicit justifications for the additional complexity added by distinguishing each category as a separate category?
+2. How is the weight of each vote determined in the proposed system, and how does it defend against fraud and manipulation?
+3. How does the proposed system decide that it has "reached quorum", and can legitimately enforce the decision that has been reached?
+4. What vectors for denial of service (at both the protocol level and social level), and how does the proposed system mitigate these?
+5. What is the proposed path to adoption of the proposed system?
+6. What "transitionary" periods does the solution include, and what are the justifications for them? Or, conversely, what is the justification for *not* having a transitionary period?
+7. What is an estimated timeline and budget for implementation of the proposed system?
+8. Who is going to implement the proposed solution, and how will it be paid for?
+
+## **Open Questions**
+
+Below are a set of philosophical questions applicable across a broad range of solutions that are being discussed by the community:
+
+1. Are there any "fundamental rights" of ecosystem participants that should be protected by the governance system?
+2. Must any and all voting related to governance matters occur directly on the layer 1 ledger?
+3. What role, if any, do the founding entities play after governance is adopted?
+4. What kinds of community standards are needed outside of the scope of a specific solution at the ledger layer?
+5. What kinds of incentives (or disinsentives) should we use to align behavior with our goals? Should we try to compensate actors for the time and effort they put in, or does that create corrupt and misaligned incentives of their own?
+6. Is the ability to delegate overall more helpful or harmful? Does the risk of "popularity" based delegation (as opposed to trust / expertise based delegation) dilute the votes of others more than it increases inclusivity?
```

Comment:
Does trust / expertise based delegation result in centralization that subjects Cardano to similar 'Cantillon effect' seen in fiat systems?